framework/db/ActiveQuery.php

```diff
@@ -438,7 +438,7 @@ private function buildJoinWith()
 
         $model = new $this->modelClass;
         foreach ($this->joinWith as $config) {
-            list ($with, $eagerLoading, $joinType) = $config;
+            [$with, $eagerLoading, $joinType] = $config;
```

Comment:
change line above to `foreach ($this->joinWith as [$with, $eagerLoading, $joinType] ) {`?